rust-runtime/inlineable/src/event_stream_receiver.rs

```diff
@@ -0,0 +1,30 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+use aws_smithy_http::event_stream::Receiver;
+use aws_smithy_runtime_api::box_error::BoxError;
+
+#[derive(Debug)]
+#[non_exhaustive]
```

Comment:
non_exhaustive isn't needed since there's a private field

Reply:
Fixed in ad8bdb8.